version: '3'

services:
  app:
    image: linuxserver/calibre-web@sha256:6bef60dc4a64b2f55b4010a2bb7d5a03017918eb76db8e430c42b8af4f7cefb3
    environment:
      - TZ=Europe/London
    volumes:
      - /var/volumes/calibre-web:/config
      - /var/volumes/nextcloud/nextcloud/data/dhaarhoff/files/calibre-library:/books:ro
    networks:
      - traefik
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.calibre-web.rule=Host(`books.rknt.de`)"
      - "traefik.http.routers.calibre-web.entrypoints=websecure"
      - "traefik.http.routers.calibre-web.tls.certresolver=myresolver"
    restart: unless-stopped

networks:
  traefik:
    external: true
